hax-lib-macros/types/src/lib.rs

```diff
@@ -54,6 +54,8 @@ pub enum AssociationRole {
     ProcessWrite,
     ProcessInit,
     ProtocolMessages,
+    PVConstructor,
+    PVHandwritten,
```

Comment:
Do you actually need constructors both in AssociationRole and in AttrPayload?
`AssociationRole` defines the nature of the relation of two linked items, e.g. a function `f` is linked to `f_precondition` (via a UUID) and the nature of their relation is "Requires" (I need to write a comment for that :sweat_smile:)

Reply:
I see, yes, then `AssociationRole` is not necessary at all, since there are no associated items.